pkg/apis/eventing/v1alpha1/channel_types.go

```diff
@@ -101,6 +101,10 @@ type ChannelStatus struct {
 	// +patchMergeKey=type
 	// +patchStrategy=merge
 	Conditions duckv1alpha1.Conditions `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+
+	// Raw is status unique to each ClusterChannelProvisioner.
+	// +optional
+	Raw *runtime.RawExtension `json:"raw,omitempty"`
```

Comment:
Seeing this without an update to the CRD yaml reminds me: we should add an OpenAPI schema to our CRDs, similar to the [ones in eventing-sources](https://github.com/knative/eventing-sources/blob/master/config/crds/sources_v1alpha1_containersource.yaml#L20).